malariagen_data/anopheles.py

```diff
@@ -7522,7 +7562,7 @@ def plot_ihs_gwss(
         gap_scale: ihs_params.gap_scale = ihs_params.gap_scale_default,
         include_edges: ihs_params.include_edges = True,
         use_threads: ihs_params.use_threads = True,
-        min_cohort_size: base_params.min_cohort_size = 15,
+        min_cohort_size: Optional[base_params.min_cohort_size] = 15,
         max_cohort_size: base_params.max_cohort_size = 50,
```

Comment:
Suggest defining defaults on `ihs_params` as above.

Also, `max_cohort_size` type should be `Optional[base_params.max_cohort_size]` here?